### PR TITLE
chore: change env vars for react []

### PR DESCRIPTION
**Pull request by kdamball**
## Purpose
Fixing slack app URL by removing CRA prefix.

## Approach
For some reason build process was linking the URL as:

```js
const backendURL = {
   REACT_APP_SLACK_BACKEND_BASE_URL: "http://someURL"
}.REACT_APP_BACKEND_BASE_URL
```
Note the difference between the two property names. Any [`REACT_APP_` prefix will be exposed in the build process](https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env). This avoids the whole problem

Also adding a cleaner and more efficient deployment process.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -76,14 +43,68 @@ jobs:
...
+          name: Install awscli
+          command: |
+            sudo apt-get update
+            sudo apt-get install python3-pip python3-dev
+            sudo pip3 install awscli
+      - run:
+          name: Build
+          command: |
+                REACT_APP_BACKEND_BASE_URL=$PROD_APP_SLACK_BACKEND_BASE_URL \
+                REACT_APP_SLACK_CLIENT_ID=$REACT_APP_SLACK_CLIENT_ID \
```
Comment (colomolo) [inline .circleci/config.yml:95]: Maybe also replace these client id vars to avoid the same problems?

**Review by ivo-toby — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (kdamball) [reply .circleci/config.yml:95]: These two are the same but will change them just in case 😅

File: .circleci/config.yml [hunk 2]
```diff
@@ -76,14 +43,68 @@ jobs:
...
+            sudo pip3 install awscli
+      - run:
+          name: Build
           command: |
-            if [ "$CIRCLE_BRANCH" != "master" ]; then
-                circleci-agent step halt
-            fi
-      - publish
-      - deploy
+                REACT_APP_BACKEND_BASE_URL=$PROD_APP_SLACK_BACKEND_BASE_URL \
```
Comment (colomolo) [inline .circleci/config.yml:94]:
```suggestion
                REACT_APP_BACKEND_BASE_URL=$APP_SLACK_BACKEND_BASE_URL_PROD \
```
for consistency.

**Review by colomolo — APPROVED** [review]
Looks good! Just have another nit.

File: .circleci/config.yml [hunk 3]
```diff
@@ -76,14 +43,68 @@ jobs:
...
-      - deploy
+                REACT_APP_BACKEND_BASE_URL=$APP_SLACK_BACKEND_BASE_URL_PROD \
+                REACT_APP_SLACK_CLIENT_ID=$SLACK_CLIENT_ID_PROD \
+                npm run build
+      - run:
+          name: Deploy apps to prod
+          command: |
+            STATIC_S3_BASE="s3://cf-apps-static/apps" \
+                STATIC_JIRA_S3_BASE="s3://cf-apps-jira" \
+                REACT_APP_BACKEND_BASE_URL=$PROD_APP_SLACK_BACKEND_BASE_URL \
```
Comment (colomolo) [inline .circleci/config.yml:102]:
```suggestion
                REACT_APP_BACKEND_BASE_URL=$APP_SLACK_BACKEND_BASE_URL_PROD \
```
and another one.

**Review by vikaskumr — APPROVED** [review]

**Pull request merged**